spec/lib/data_update_scripts/profile_website_url_format_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210722135549_profile_website_url_format.rb",
+)
+
+describe DataUpdateScripts::ProfileWebsiteUrlFormat do
+  let(:nil_profile) { create(:profile, website_url: nil) }
+  let(:empty_profile) { create(:profile, website_url: "") }
+  let(:valid_profile) { create(:profile, website_url: "https://www.example.com") }
+
+  let(:invalid_profile) do
+    create(:profile).tap do |profile|
+      profile.update_column(:website_url, "www.example.com")
```

Comment:
is there an easier way to bypass validations for factory generated objects?

Reply:
How about this?

```ruby
let(:invalid_profile) do
  profile = build(:profile, website_url: "www.example.com")
  provile.save(validate: false)
end
```

Reply:
that is, in fact, just what I wanted to say. Perfect!